processing/src/main/java/io/druid/query/filter/DruidFloatPredicate.java

```diff
@@ -22,9 +22,32 @@
 /**
  * FloatPredicate is only supported in Java 8+, so use this to avoid boxing when a float predicate is needed.
```

Comment:
`FloatPredicate` doesn't exist in JDK. Even if it existed, this comment is stale because now the Druid's interface has two methods.

Reply:
Please add "Note: this is not a `{@link PublicApi}` or an `{@link ExtensionPoint}` of Druid."